src/test/kotlin/com/forgerock/uk/openbanking/tests/functional/payment/domestic/payments/legacy/LegacyCreateDomesticPaymentTest.kt

```diff
@@ -116,6 +116,10 @@ class LegacyCreateDomesticPaymentTest(val tppResource: CreateTppCallback.TppReso
         assertThat(result).isNotNull()
         assertThat(result.data).isNotNull()
         assertThat(result.data.consentId).isNotEmpty()
+        if (consent.data.charges.isNullOrEmpty())
```

Comment:
Why that condition? it's confuse me.

Reply:
This condition is regarding the issue https://github.com/SecureBankingAccessToolkit/SecureBankingAccessToolkit/issues/488, to be easier to test the solution.